pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -390,3 +393,37 @@ type EC2SDConfig struct {
 	// +optional
 	Filters []*EC2Filter `json:"filters"`
 }
+
+// GCESDConfig allow retrieving scrape targets from GCE instances.
+// The private IP address is used by default, but may be changed to the public IP address with relabeling.
+// See https://prometheus.io/docs/prometheus/latest/configuration/configuration/#gce_sd_config
+// Note: Users should update the Prometheus CR's spec.secret to include the Secret
```

Comment:
(suggested text)

The GCE service discovery will load the Google Cloud credentials from the file specified by the  `GOOGLE_APPLICATION_CREDENTIALS` environment variable.
A pre-requisite for using `GCESDConfig` is that a Secret containing valid Google Cloud credentials is mounted into the Prometheus or PrometheusAgent pod via the `.spec.secrets` field and that the `GOOGLE_APPLICATION_CREDENTIALS` environment variable is set to `/etc/prometheus/secrets/<secret-name>/<credentials>`.